footer.css

```diff
@@ -0,0 +1,27 @@
+.stopka {
+	padding: 0 20px;
+    font-size: 15px;
+    display: flex;
+    align-items: center;
+   justify-content: space-between;
+    color: #A67D4A;
+	height: 50px;
+	text-transform: capitalize;
+	background: linear-gradient(left,#ffffff, rgb(204, 204, 204));
+	background: -moz-linear-gradient(left,#ffffff, rgb(204, 204, 204));
+	background: -o-linear-gradient(left,#ffffff, rgb(204, 204, 204));
+	background: -webkit-linear-gradient(left,#ffffff, rgb(204, 204, 204));
+}
+.stopka1 {
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+}
+.tekst1 {
+	padding: 0 10px;
+}
+.tekst2:hover {
```

Comment:
Czy jest jakaś różnica w działaniu `cursor: pointer` dla samego `tekst2` i `tekst2:hover`? Kursor zmienia się dopiero po najechaniu na element, wcześniej nie jest to fizycznie możliwe. Utrzymuje się, że dla `cursor` najbardziej odpowiedni będzie selektor bez pseudoklasy `hover`.